Makefile-ponyc

```diff
@@ -164,7 +164,10 @@ endif
 
 ifneq (,$(filter $(link),static llvm-static))
   LLVM_LINKING =--link-static
-  LINKER_FLAGS += -static-libstdc++
+  ifeq ($(OSTYPE),linux)
+    # definitely not for freebsd, do we need for windows?
```

Comment:
any input @kulibali? I'd prefer to get this right when the PR goes in.

Reply:
perhaps, for now, the best check would be "not FreeBSD" as we know it doesn't work on FreeBSD.

Reply:
Not applicable to windows; on Unix I'm not actually sure whether CMake is linking the standard library statically; I'll need to double-check that.
But this change looks fine to me for the time being.